app/src/views/GrantRound.vue

```diff
@@ -329,19 +336,28 @@ function useGrantRoundDetail() {
     // contributionAmount must have the right number of decimals and be hexed
     const contributionAmount = parseUnits(amount, grantRound.value.matchingToken.decimals);
 
+    let txIndex = 1;
+    let txsNeeded = 1;
```

Comment:
is this needed ? it seems like it's always `txsNeeded = txIndex + 1`

Reply:
if you need to approve (line `343`) txsNeeded will be different